tests/api.py

```diff
@@ -45,8 +53,8 @@ def db_init(app):
 
     app.oauth2 = OAuth2Client(**app.config['OAUTH2'])
 
-    app.logger.info('Initializing Signpost driver')
-    app.signpost = SignpostClient(
+    app.logger.info('Initializing Indexd driver')
+    app.signpost = IndexClient(
```

Comment:
mb rename it to `app.indexd` then?

Reply:
I think this will cause too much trouble with what @Avantol13 is doing.

Reply:
yeah, we gotta merge branches faster, we are working too close now all of a sudden

Reply:
I think we will make a separate task for this. I think it's quickly becoming too much trouble for what it's worth currently.